src/lando/tests/test_version.py

```diff
@@ -1,25 +1,22 @@
-from django.conf import settings
 from django.core.management import call_command
 
 import pytest
 
-def test_version():
-    # The version file may or may not exist in the testing environment.
-    # We'll explicitly remove it so that we're in a known state, then
-    # re-generate it and test against it.
-    version_file = settings.BASE_DIR / "version.py"
-    version_file.unlink(missing_ok=True)
-
-    # We should not be able to import it after it's been removed.
-    with pytest.raises(ImportError):
-        from lando.version import version
 
+@pytest.fixture
+def generate_version_file():
```

Comment:
Can we change this to `lando_version` and then `return version` from the fixture?

Reply:
Done.